### DISCUSSION TITLE
Alias with Automatic or Custom Disabling time.

### DISCUSSION THREAD

**Discussion by swapnil2905** [simple-login/app#637] [Feature Requests]
Hello Team Simple-login, 
The Feature i am requesting is to have an option which will provide an automatic disabling of an alias after a custom interval(5minutes or so) this feature will allow the users to prevent spam coming to their inboxes from the same email. 
I know we can manually disable an alias, but sometimes we forget about it, and then we end up receiving the bunch of spam emails.
My suggestion is to have a small alias disabling timer besides the new alias creation button.
Hope this feature gets implemented !

**Comment by nguyenkims** [comment]
That's a great idea! This way SimpleLogin can achieve what temporary email services are providing.

**Reply by swapnil2905** [reply]
Thanks! Happy to contribute to this Amazing Application. 
I love this Application.

**Reply by swapnil2905** [reply]
Hello Son @nguyenkims  Any update on this feature when can I expect this to be implemented, I really need this feature!

**Reply by nguyenkims** [reply]
@swapnil2905 unfortunately we don't have any ETA on when this feature is implemented.

**Reply by swapnil2905** [reply]
@nguyenkims Hoping this won't be a big deal to implement this feature, hoping to see this feature by end of this year.
